test/tests/imageserver/test_image_service_system.py

```diff
@@ -0,0 +1,319 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+This test validate the system level function of RackHD image-service. This test include VMware ESXi install and node
+rediscover.It use image server to store ESXi image and microkernel used for RackHD discovery.
+You need put an config file in the /config directory
+'''
+
+import os
+import sys
+import time
+import flogging
+import random
+import json
+import fit_common
+import pexpect
+import unittest
+import test_api_utils
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+
+try:
+    FILE_CONFIG = json.loads(open(fit_common.CONFIG_PATH + "fileserver_config.json").read())
+except BaseException:
+    logs.error(
+        "**** Global Config file: " + fit_common.CONFIG_PATH + "FILE_CONFIG.json" +
+        " missing or corrupted! Exiting....")
+    sys.exit(255)
+
+
+@attr(all=True, regression=False, smoke=False)

```

Comment:
add a group here for imageservice=True  (or something good).
That way we can set up a jenkins job to target this suite  specifically.